src/core/server/base.js

```diff
@@ -30,7 +30,20 @@ const version = path.join(config.get('basePath'), 'version.json');
 const isDeployed = config.get('isDeployed');
 const isDevelopment = config.get('isDevelopment');
 
-const errorString = 'Internal Server Error';
+const errorPageText = {
+  401: 'Unauthorized',
```

Comment:
As it stands this is surfacing a 401. I need to check with mat why we have a 401 on the api for an addon detail which is surfaced as a 404 on the current site.